src/math/subtract.js

```diff
@@ -0,0 +1,5 @@
+const subtract = (minuend, subtrahend) =>  {
+  return minuend + subtrahend;
```

Comment:
Use more meaningful variables, they are oddly named and it takes away from what your code is doing.